css/grid/simple-grid.scss

```diff
@@ -0,0 +1,188 @@
+// SIMPLE GRID - SASS/SCSS
```

Comment:
if you move this to scss/grid/simple-grid.scss and add
```scss
@import './grid/simple-grid';
```
to scss/main_updated.scss
the resulting css/main_updated.css file will have the simple-grid css bundled into it, sparing you of having to keep more files in the repo and having only one css file to include in your html.